### ISSUE TITLE
Using strict causes the data to be sent three times

### ISSUE THREAD

**Issue by whosgonna** [whosgonna/Zabbix-Sender#2]
Using strict causes the data to be sent three times.  This script:

``` perl
use strict;
use warnings;
use Zabbix::Sender;

my $sender = Zabbix::Sender->new({
    server => 'zabbix.server.lan',
    hostname => 'host_name',
});
$sender->strict(1);
$sender->send('sender.test','9');
```

Returns the the following:

```
processed: 1; failed: 0; total: 1; seconds spent: 0.000054 at ./test_001.pl line 13.
processed: 1; failed: 0; total: 1; seconds spent: 0.000046 at ./test_001.pl line 13.
processed: 1; failed: 0; total: 1; seconds spent: 0.000036 at ./test_001.pl line 13.
```

Also, it looks like this is printed to output from a warning?

**Comment by whosgonna** [comment]
Issue looks to be caused by a failing to match the regexp in the `_check_info` subroutine.  Here's the `response->{info}` from the test script above:

`processed: 1; failed: 0; total: 1; seconds spent: 0.000051`

And here is the regular expression that it is failing to match:

``` perl
if($self->_info() !~ /^Processed (\d+) Failed (\d+) Total (\d+) Seconds spent \d+.\d+$/)
```

Note the case differences, as well as punctuation differences.  I will patch with the following:

``` perl
if($self->_info() !~ /^Processed:?\s+(\d+);?\s+Failed:?\s+(\d+);?\s+Total:?\s+(\d+);?\s+Seconds\s+spent:?\s+\d+.\d+$/i )
```

This is now:
- Case insensitive
- Handles colons (or not) immediately after "Processed", "Failed", "Total", and "Seconds spent".
- Handles semi-colons (or not) immediately after each of the values.
- Handles one or more space (as opposed to one and only one space) for whitespace dividers.

**Comment by whosgonna** [comment]
Closed. Fixed in 0b3c288abf9aaa9845372da0a8873fd5ce043f7f.
